File: 1/solution.py
import sys

sums = [
    sum(map(int, ea.split("\n")))
    for ea in open(sys.argv[1], "r").read().strip().split("\n\n")
]

# part 1
print(max(sums))

# part 2
print(sum(sorted(sums, reverse=True)[:3]))

File: 1/solution.jl
sums = map(
    x -> sum(
        map(y -> parse(Int64, y), x)
    ),
    map(
        z -> split(z, "\n"),
        split(chomp(read(open(ARGS[1], "r"), String)), "\n\n")
    )
)

# part 1
println(maximum(sums))

# part 2
println(sum(sort(sums, rev=true)[begin:3]))
